CMakeLists.txt

```diff
@@ -33,7 +33,12 @@ list(APPEND INCLUDE_DIRS
 
 list(APPEND SOURCE_FILES "asm/CortexContextSwitch.s")
 list(APPEND SOURCE_FILES "nrfx/mdk/system_${DEV}.c")
-list(APPEND SOURCE_FILES "nrfx/mdk/gcc_startup_${DEV}.S")
+
+if(EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/nrfx_mods/mdk/gcc_startup_${DEV}.S")
+    list(APPEND SOURCE_FILES "nrfx_mods/mdk/gcc_startup_${DEV}.S")
+else()
+    list(APPEND SOURCE_FILES "nrfx/mdk/gcc_startup_${DEV}.S")
+endif()
```

Comment:
Ah, good catch, I forgot yesterday that there could be other nrf52xxx targets here.